app/model/dto/models.go

```diff
@@ -18,11 +21,9 @@ type Recipe struct {
 	Author    uuid.UUID
 	Portions  int
 	Tags      []string
+    Calories  int
```

Comment:
pytanie: kalorie na porcje czy kalorie całego przepisu?

Reply:
Możesz to zastąpić funkcją modelu która obliczy kalorie na bazie składników bo w modelu mamy te Infredient i IngredientAmount z kcal, jak to rozłączymy i będziemy ustawiać osobno to może się zdarzyć że nam się to rozjebie i spójność pójdzie w pisduuuu

Reply:
możesz też zrobić pustą funkcję która zwróci jakąs stałą wartość, jebnąć todosa i dodać taska ;)

Reply:
po namyśle zostaw jak jest

Reply:
hahaha cieszę się że pod moją nieobecność doszedłeś do porozumienia